src/ts/components/fullscreentogglebutton.ts

```diff
@@ -28,17 +28,34 @@ export class FullscreenToggleButton extends ToggleButton<ToggleButtonConfig> {
       }
     };
 
+    let fullscreenAvailableHandler = () => {
+      if (player.isViewModeAvailable(player.exports.ViewMode.Fullscreen)) {
+        this.show();
+      } else {
+        this.hide();
+      }
+    };
```

Comment:
It seems a bit counter-intuitive to me, that a `fullscreenAvailableHandler` needs to check, whether fullscreen is available or not, as the name suggests that it is. Is this check actually necessary? If so, I think `fullscreenAvailabilityChangedHandler` would be a more expressive name.

Reply:
At the moment the naming matches in my opinion. As the availability can't change (yet - pr incoming) it does exactly what it says.

This will be changed soon.